Assets/Scripts/Scenes/InGame/Player/Item/Compass/CompassMove.cs

```diff
@@ -0,0 +1,97 @@
+using System.Collections;
+using System.Collections.Generic;
+using Unity.VisualScripting;
+using UnityEngine;
+using DG.Tweening;
+
+public class CompassMove : MonoBehaviour
+{
+    private GameObject _escapePoint;
+    private GameObject[] _Enemys;
+    private float distance;
+    private bool _isEnabled = true;
+    [Header("�j�̉�]���x")]
+    [SerializeField] private float _rotateSpeed;
+    [Header("�j�̉�]����(�̏᎞)")]
+    [SerializeField] private float _brokeRotateCycle;
+    [Header("�R���p�X�����鋗��")]
+    [SerializeField] private float _unusableDistance;
+    private void Start()
+    {
+        _escapePoint = GameObject.Find("escapePoint(Clone)");
```

Comment:
```suggestion
        _escapePoint = GameObject.FindGameObjectsWithTag("EscapePoint");
```
現状のままだと、オブジェクトの名前が変わった際にエラーが起きるため別の方法で取得してください。
※オブジェクトの名前は特に変わりやすいので